modules/transport-netty4/src/main/java/org/elasticsearch/transport/netty4/ESLoggingHandler.java

```diff
@@ -104,6 +104,8 @@ else if (readableBytes >= TcpHeader.HEADER_SIZE) {
                             try (ThreadContext context = new ThreadContext(Settings.EMPTY)) {
                                 context.readHeaders(in);
                             }
+                            // now we decode the features
+                            in.readStringArray();
```

Comment:
shouldn't we have a version protection here?

Reply:
Good catch. I pushed 0b39ce997889035d44100c3a7cb8394eee155d76.